tests/scripts/run-xunit-perf.cmd

```diff
@@ -102,16 +103,14 @@ setlocal
   rem optionally generate results for benchview
   if exist "%BENCHVIEW_PATH%" (
     call :generate_results_for_benchview || exit /b 1
-  ) else (
-    type "%XMLOUT%" | findstr /i /c:"test name"
-  )
+  ) 
 
   rem Save off the results to the root directory for recovery later in Jenkins
   IF EXIST "Perf-%BENCHNAME%.xml" (
```

Comment:
The condition testing if the expected output file exist would fail because it will not match the actual output file (testing for *Perf-LinkBench.xml* but generated *Perf-CoreFx-LinkBench.xml*). I would change the `IF EXIST` and `COPY` commands to check and copy `"Perf-*%BENCHNAME%.xml"` and `"Perf-*%BENCHNAME%.etl"`